tests/test_template.py

```diff
@@ -181,18 +181,6 @@ def test_get_template_details_without_upload(self):
 
         assert boto_parameter == {"TemplateBody": sentinel.body}
 
-    def test_body_with_json_template(self):
```

Comment:
Essentially this test would always fail because, with this change, the template_body is no longer valid JSON, no matter the input. It is however valid YAML.